app/src/test/java/edu/uc/kovaciad/slicetracker/ExampleUnitTest.kt

```diff
@@ -1,17 +0,0 @@
-package edu.uc.kovaciad.slicetracker
-
-import org.junit.Test
-
-import org.junit.Assert.*
-
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * See [testing documentation](http://d.android.com/tools/testing).
- */
-class ExampleUnitTest {
-    @Test
-    fun addition_isCorrect() {
-        assertEquals(4, 2 + 2)
-    }
-}
```

Comment:
This class had become redundant. The addition_isCorrect() test was recreated in the DataUnitTest class as twoPlusTwo(). With no other content present, it made sense to remove the class altogether and write all relevant tests in the DataUnitTest class.